client/src/constants/analysis_constants.js

```diff
@@ -0,0 +1,23 @@
+const HEADLINE_PLACEHOLDER =
```

Comment:
Put all your mock/placeholder stuff in one place please.  If you scatter it all over your code there's a risk you'll forget one (or whoever has to come in and change this code after you will miss one).